src/ncp/ncp_base.cpp

```diff
@@ -4212,44 +4229,31 @@ otError NcpBase::SetPropertyHandler_PHY_ENABLED(uint8_t header, spinel_prop_key_
                        &value
                    );
 
-    if (parsedLength > 0)
+    VerifyOrExit(parsedLength > 0, errorCode = OT_ERROR_PARSE);
+
+    if (value == false)
     {
-        if (value == false)
+        // If we have raw stream enabled stop receiving
+        if (mIsRawStreamEnabled)
         {
-            // If we have raw stream enabled stop receiving
-            if (mIsRawStreamEnabled)
-            {
-                otLinkRawSleep(mInstance);
-            }
-
-            errorCode = otLinkRawSetEnable(mInstance, false);
+            otLinkRawSleep(mInstance);
         }
-        else
-        {
-            errorCode = otLinkRawSetEnable(mInstance, true);
 
-            // If we have raw stream enabled already, start receiving
-            if (errorCode == OT_ERROR_NONE && mIsRawStreamEnabled)
-            {
-                errorCode = otLinkRawReceive(mInstance, mCurReceiveChannel, &NcpBase::LinkRawReceiveDone);
-            }
-        }
+        errorCode = otLinkRawSetEnable(mInstance, false);
     }
     else
     {
-        errorCode = OT_ERROR_PARSE;
-    }
+        errorCode = otLinkRawSetEnable(mInstance, true);
 
-    if (errorCode == OT_ERROR_NONE)
-    {
-        errorCode = HandleCommandPropertyGet(header, key);
-    }
-    else
-    {
-        errorCode = SendLastStatus(header, ThreadErrorToSpinelStatus(errorCode));
+        // If we have raw stream enabled already, start receiving
```

Comment:
diff looks rather weird... All is happening here is the back-indent of the code (due to `VerifyOrExit`) at the top.
And replcating of the code block below with `SendSetPropertyResponse()` at `exit`:
```
if (errorCode == OT_ERROR_NONE)
{
    errorCode = HandleCommandPropertyGet(header, key)
}
else
{
   errorCode = SendLastStatus(header, ThreadErrorToSpinelStatus(errorCode));
}
```